Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -3874,8 +3871,180 @@ def get_detection_for_incident_command(incident_id: str) -> CommandResults:
                           raw_response=detection_res)
 
 
+def cs_falcon_spotlight_search_vulnerability_command(args: dict, filter_operator='AND') -> CommandResults:
+    """
+        Get a list of vulnerability by spotlight
+        : args: filter which include params or filter param.
+        : return: a list of vulnerabilities according to the user.
+    """
+    if not args:
+        raise DemistoException('Please add a at least one filter argument')
+    input_arg_dict = {'aid': argToList(args.get('aid')),
+                      'cve.id': argToList(args.get('cve_id')),
+                      'cve.severity': argToList(args.get('cve_severity')),
+                      'host_info.tags': argToList(args.get('tags')),
+                      'status': argToList(args.get('status')),
+                      'host_info.platform_name': args.get('platform_name'),
+                      'host_info.groups': argToList(args.get('host_group')),
+                      'host_info.product_type_desc': argToList(args.get('host_type')),
+                      'last_seen_within': args.get('last_seen_within'),
+                      'suppression_info.is_suppressed': args.get('is_suppressed')}
+    if not args or input_arg_dict == {}:
+        raise DemistoException('Please add a at least one filter argument')
+    url_filter = '{}'.format(str(args.get('filter', '')))
+    op = ',' if filter_operator == 'OR' else '%2B'
+    # In Falcon Query Language, '+' (after decode '%2B) stands for AND and ',' for OR
+    # (https://falcon.crowdstrike.com/documentation/45/falcon-query-language-fql)
+    for key, arg in input_arg_dict.items():
+        if arg:
+            if url_filter:
+                url_filter += '%2B'
+            if type(arg) is list:
+                url_filter += f'{key}:[\'' + "','".join(arg) + '\']'
+            else:
+                # All args should be a list. this is a fallback
+                url_filter = "{url_filter}{operator}{inp_arg}:'{arg_val}'".format(url_filter=url_filter, operator=op,
+                                                                                  inp_arg=key, arg_val=arg)
+    url_facet = '&facet=cve'
+    if argToBoolean(args.get('display_remediation_info')):
+        url_facet += "&facet=remediation"
+    if argToBoolean(args.get('display_evaluation_logic_info')):
+        url_facet += "&facet=evaluation_logic"
+    if argToBoolean(args.get('display_host_info')):
+        url_facet += "&facet=host_info"
+    # The url is hardcoded since facet is a parameter that can have serval values, therefore we can't use a dict
+    endpoint_url = '/spotlight/combined/vulnerabilities/v1?' + 'filter=' + \
+        url_filter + url_facet + "&limit=" + args.get('limit', '50')
+    vulnerability_response = http_request('GET', endpoint_url)
```

Comment:
You need to split the command to request and the command itself. 
Don't do it all together.
The command should handle only the response

Reply:
added a request function